tests/mock_vllm_model.py

```diff
@@ -318,3 +335,105 @@ def forward(
                 kv_cache=kv_cache,
                 prompt_lens=prompt_lens,
             )
+
+
+class RawStatLogger(StatLoggerBase):
```

Comment:
Is this Class different from the one in `vllm-tt-metal-llama3-70b/src/logging_utils.py`? Can you explain the difference?

Reply:
It is the same. It is duplicated since due to the current directory naming structure with the hyphens it is not possible to import `logging_utils`. Is there a better way to go about this?

Reply:
Ok makes sense. Can you add a header to the file then that you think should be removed after the refactor?

Maybe something with a `# TODO:` identifier so that we can keep track of it?